src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -469,4 +470,30 @@ private String evaluateMacro(String template) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    private void pullRequestActionable(List<PotentialAction> paList, List<Fact> factsList) {
+        Job job = run.getParent();
+        ItemGroup parent = job.getParent();
+        if (parent instanceof MultiBranchProject) {
+            BranchProjectFactory projectFactory = ((MultiBranchProject) parent).getProjectFactory();
+            if (projectFactory.isProject(job)) {
+                Branch branch = projectFactory.getBranch(job);
+                SCMHead head = branch.getHead();
+
+                if (head instanceof ChangeRequestSCMHead) {
+                    ObjectMetadataAction oma = branch.getAction(ObjectMetadataAction.class);
+                    if (oma != null) {
+                        String urlString = oma.getObjectUrl();
+                        PotentialAction viewPRPotentialAction = new PotentialAction("View PR", urlString);
+                        paList.add(viewPRPotentialAction);
+                        factsList.add(new Fact(head.getPronoun() + " Title", oma.getObjectDisplayName()));
```

Comment:
Pronoun fetches each SCM's defined pronoun for a change request, so for github, it would be pull request.